tests/test_scheduler.py

```diff
@@ -1572,7 +1572,7 @@ def test_scheduler_nested_type_enrichment(frontend, config):
     assert calls[1].name.parent.parent.parent.type.dtype.name == 'third_type'
     assert isinstance(calls[1].name.parent.parent.parent.type.dtype.typedef, TypeDef)
 
-    inline_calls = FindInlineCalls().visit(driver.body)
+    inline_calls = FindInlineCalls(unique=False).visit(driver.body)
```

Comment:
This seems entirely unrelated to the PR. Is there a deeper reason for this?

Reply:
Now the tests pass even without the `unique=False` argument 🤔 Don't know why I needed this at the time, but I've gotten rid of it.